spec/features/workflow/schedule_set_datetime_passes_spec.rb

```diff
@@ -5,6 +5,7 @@
     given_there_is_an_edition_with_past_scheduled_publishing_datetime
```

Comment:
I'm wondering if this spec is covered by the ones for changing the date?

Reply:
I don't think so because this spec is testing that the user is unable to click on the schedule button if the scheduled_publishing_datetime is in the past.